extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -166,9 +177,27 @@ export const ManageAssetRows = ({
     }
   }, [submitStatus, assetSubmitting, setErrorAsset, dispatch]);
 
+  const isBlockedDomain = (domain: string) => {
+    const found = blockedDomains.domains.filter((b) => b.domain === domain);
+    return found.length > 0;
+  };
+
+  const handleBlockedDomain = (assetData: AssetRowData) => {
+    dispatch(showBlockedDomainWarning());
```

Comment:
related to using `slideupModal`, but wondering if you even need to run this through redux. Seems like all the state changes are happening right in this component

Reply:
hmm good point technically it doesn't need to be in redux 🤔 . If alright with you I'd like to finish the scam asset work (should be 1 more major PR), and if it is simpler not in redux after that work then I can remove. They'll be 1 more warning screen, and possibly other data I'm not thinking of now that could be put into redux and it would be nice to keep that data together I think

Reply:
Good plan. No need to optimize if there's still more screens to add 👍